conf/application.conf

```diff
@@ -136,4 +136,6 @@ mongodb {
   uri = "mongodb://localhost:27017/eeitt-admin-frontend"
 }
 
-sdes.delete.olderThan = 24
\ No newline at end of file
+sdes.delete.olderThan = 24
+
+diff.timeout= "60 seconds"
```

Comment:
I think this is too much time for people to wait. I think setting it to 15 seconds may be better.